### PR TITLE
Skunkworks - AtlasCLI copy and docs (Part 1)

### PR DESCRIPTION
**Pull request by blva**
## Proposed changes

Process: 
1. Each person took a set of commands to review
2. Each one wrote a set of suggestions for their commands
3. Each one reviews the docs by the other ones
4. Bianca writes the updates 

[[copy] atlas deployments setup by Corry](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/6c6e5d74a93816bf29c539e5bbc21a3495d21db9)
[6c6e5d7](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/6c6e5d74a93816bf29c539e5bbc21a3495d21db9)
[atlas config describe by Sarah](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/f52e90444562b3f41cdd37ae312f2bc4158582b7)
[f52e904](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/f52e90444562b3f41cdd37ae312f2bc4158582b7)
[flag usage by Corry](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/12bb915403075a1205466442c0330d043d5e76af)
[12bb915](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/12bb915403075a1205466442c0330d043d5e76af)
[atlas config set by Carrie](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/9d2be4104794925c920dc8ff62d30b175dc27c8a)
[9d2be41](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/9d2be4104794925c920dc8ff62d30b175dc27c8a)
[atlas accesslists create by Carrie](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/f890158b0cca568b740c3a92f0eadff946f5b40d)
[f890158](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/f890158b0cca568b740c3a92f0eadff946f5b40d)

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by fmenezes — APPROVED** [review]
make sure to get a review from docs

**Review by wtrocki — APPROVED** [review]

File: internal/usage/usage.go
Comment (sarahsimpers) [inline internal/usage/usage.go:10]: Did we mean to change this?

File: internal/usage/usage.go
Comment (sarahsimpers) [inline internal/usage/usage.go:450]: I haven't seen the original copy suggestion, but I think this makes more sense
```suggestion
	UpdateWarning                             = "Flag that indicates whether to replace preexisting data."
```

File: internal/usage/usage.go
Comment (sarahsimpers) [inline internal/usage/usage.go:417]: I believe the comment that was removed here is required to pass linting due to the use of the word "secrets"? But not sure

**Review by sarahsimpers — COMMENTED** [review]
Question for @corryroot who did the copy review for usage.go... how do we want to determine which descriptions we start with "Flag that...." since they are all flags when used in a CLI (as opposed to the API)? Are we using it here only when the flag itself does the enabling/disabling (so you specify "--flag" to enable something, not "--flag true"?) Just making sure I understand. If it's the latter, I'm not sure whether all of the replacements we made for ones that said "if true" or "if false" make sense, but will defer to you on this. Thanks!

File: internal/usage/usage.go
Comment (blva) [reply internal/usage/usage.go:10]: no, thanks!

File: internal/usage/usage.go
Comment (blva) [reply internal/usage/usage.go]: updated! keeping this for history

File: internal/usage/usage.go
Comment (corryroot) [inline internal/usage/usage.go:247]: **Question:**

@dianchenghu and @blva, are all booleans passed as simple flags, or do users actually have to specify "true" and "false"? See @sarahsimpers comment. If so, can we have a consistent behavior.

File: internal/usage/usage.go
Comment (corryroot) [inline internal/usage/usage.go:304]: **Typo:**

```suggestion
	SkipSampleData                            = "Flag that indicates whether to skip loading sample data into your MongoDB cluster."
```

File: internal/usage/usage.go
Comment (corryroot) [inline internal/usage/usage.go:429]: @dianchenghu and @blva, sorry, since this isn't a typical flag description, I think we should change this back.

**Review by corryroot — COMMENTED** [review]
@blva and @dianchenghu, I have a follow up question to @sarahsimpers's comment. I noticed a very small typo and want to undo one of the changes.

File: internal/usage/usage.go
Comment (blva) [reply internal/usage/usage.go:247]: any flag marked with `BoolVar` will be just passed as flags indeed, nobody needs to specify true, but some will have default to "true" and some will have default to "false"

**Review by sarahsimpers — DISMISSED** [review]

**Comment by sarahsimpers** [comment]
LGTM

File: internal/usage/usage.go
Comment (dianchenghu) [reply internal/usage/usage.go:247]: @corryroot  We set default values for each bool flag in code if they're not specified.
For each of the above under index create, the default is false.
So "indicates that" would probably be better for them.

**Review by corryroot — COMMENTED** [review]
@blva, I spotted just one more that we should change back. Otherwise, this LGTM!

File: internal/usage/usage.go
Comment (blva) [reply internal/usage/usage.go:450]: are you refering to only UpdateWarning?

Comment (corryroot) [reply internal/usage/usage.go:450]: Yes, back to:
```
" Passing this flag replaces preexisting data."
```

**Review by corryroot — APPROVED** [review]
LGTM!

**Pull request merged**
